testnets/holesky/docker-compose.yml

```diff
@@ -82,7 +76,7 @@ volumes:
     driver: local
 
 networks:
-  bolt-default:
+  bolt-holesky-default:
     driver: bridge
```

Comment:
You will have to add `name: bolt-holesky-default` here. Otherwise docker compose will prefix it with some bs and the name will differ in `docker-compose.pbs.yml`

Reply:
Updated this to drop all references to custom networks, we will use `host.docker.internal` as DNS and adding:
```yaml
    extra_hosts:
      - host.docker.internal:host-gateway
```
On every service that needs to communicate with others which expose a port on the host machine